core/core.js

```diff
@@ -84,48 +87,31 @@
     return errorMap[errorName] ?? [undefined, []];
   }
 
-  function processResponse(res) {
-    if (!isErr(res)) {
-      return res;
-    }
-    throw processErr(res);
-  }
-
-  // .$err_class_name is a special key that should only exist on errors
-  function isErr(res) {
-    return !!(res && res.$err_class_name);
-  }
-
-  function processErr(err) {
-    const className = err.$err_class_name;
-    const [ErrorClass, args] = getErrorClassAndArgs(className);
-    if (!ErrorClass) {
-      return new Error(
-        `Unregistered error class: "${className}"\n  ${err.message}\n  Classes of errors returned from ops should be registered via Deno.core.registerErrorClass().`,
-      );
+  function unwrapResponse(res) {
```

Comment:
Might make sense to call this `unwrapResult` since the value here is conceptually similar to a rust `Result`

Reply:
Or maybe `unwrapOpResult` to be a little more explicit.